component/otelcol/exporter/prometheus/internal/convert/convert_test.go

```diff
@@ -310,7 +310,7 @@ func TestConverter(t *testing.T) {
 			includeScopeInfo: true,
 			expect: `
 				# TYPE otel_scope_info gauge
-				otel_scope_info{name="a-name",version="a-version",something_extra="zzz-extra-value"} 1.0
+				otel_scope_info{otel_scope_name="a-name",otel_scope_version="a-version",something_extra="zzz-extra-value"} 1.0
```

Comment:
I think this is a bug fix, can you list it as such?

Reply:
@gouthamve Is this the right set of labels for when the `otel_scope_info` metric _is_ written?

Reply:
This is the right set of labels. 

See example here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/prometheus_and_openmetrics.md#instrumentation-scope